packages/data-context/src/actions/ProjectActions.ts

```diff
@@ -138,6 +138,10 @@ export class ProjectActions {
     return this.globalLaunchCount
   }
 
+  set launchCount (count) {
```

Comment:
I genuinely don't like this, but I don't see any other way to get the tests to pass for the changes I made. I have to provide a setter to reset the launch count before the tests are run. If I don't, the test will fail on a second run.